### PR TITLE
Fix home-assistant dependencies

### LINKED ISSUE #59: Home assistant from git does not start properly
When starting home assistant from git (for local dev) a few components fail to start, failing with the error:

`ImportError: libstdc++.so.6: cannot open shared object file: No such file or directory`

Full error:

image

### PR DESCRIPTION
**Pull request by abmantis**
Fixes #59

### CODE REVIEW

**Comment by Mic92** [comment]
@mergify queue

**Comment by Mic92** [comment]
@mergify unqueue

File: envs/home-assistant/shell.nix [hunk 1]
```diff
@@ -15,7 +15,9 @@ pkgs.mkShell {
   ];
   # fix homeassistant-pyozw build
   NIX_CFLAGS_COMPILE = [ "-Wno-error=format-security" ];
+  LD_LIBRARY_PATH = pkgs.lib.makeLibraryPath [ pkgs.stdenv.cc.cc ];
   buildInputs = with pkgs; [
+    gcc
```
Comment (Mic92) [inline envs/home-assistant/shell.nix:20]: Are you show about this one? 
Using `pkgs.mkShell` should already give you a compiler (gcc on Linux).

File: envs/home-assistant/shell.nix [hunk 2]
```diff
@@ -15,7 +15,9 @@ pkgs.mkShell {
   ];
   # fix homeassistant-pyozw build
   NIX_CFLAGS_COMPILE = [ "-Wno-error=format-security" ];
+  LD_LIBRARY_PATH = pkgs.lib.makeLibraryPath [ pkgs.stdenv.cc.cc ];
```
Comment (Mic92) [inline envs/home-assistant/shell.nix:18]: This one seems fine.

File: envs/home-assistant/shell.nix (hunk 1 above)
Comment (abmantis) [reply envs/home-assistant/shell.nix]: That is true! Removed it.

**Comment by Mic92** [comment]
@mergify queue

**Pull request merged**